lib/hooks/use_effect_on_init.dart

```diff
@@ -0,0 +1,8 @@
+import "package:flutter_hooks/flutter_hooks.dart";
```

Comment:
So, instead of that function, we alternatively could use default `useEffect` function always passing empty list? And since we often need to run some state related function once, on init, this wrapper is handy. Am I right? 
If yes, it is nice improvement actually. Hooks got my heart.

Reply:
exactly - we can use `useEffect` with empty dependency list everytime, but we can easily create custom hooks like this in very simple way to introduce reusability. This is very simple example of a custom hook, but custom hooks can also have a big chunks of logic too